### PR TITLE
Print nexus info

### PR DESCRIPTION
**Pull request by rustydb**
This just additionally prints the nexus RPM info when `metalid.sh` is invoked.

### CODE REVIEW

File: .github/workflows/merge-strategy.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: Merge Strategy
```
Comment (heemstra) [inline .github/workflows/merge-strategy.yml:1]: this stuff should be in a separate PR ideally

**Review by heemstra — APPROVED** [review]

File: .github/workflows/merge-strategy.yml (hunk 1 above)
Comment (rustydb) [reply .github/workflows/merge-strategy.yml:1]: forgot to rebase off main, didn't mean to include this
